common/src/main/java/com/wynntils/core/WynntilsMod.java

```diff
@@ -85,6 +86,8 @@ public static void init(String modVersion, boolean isDevelopmentEnvironment) {
         ClientCommandManager.init();
         KeyManager.init();
         CompassManager.init();
+        ScoreboardManager.init();
+
```

Comment:
Even though the list of init:s is long, this break seems pretty arbitrary. If you find some reasonable way to group the manager, go ahead, but otherwise this is just one in a long list of bootstrapping modules.

Reply:
Sure, I can just remove the newline. If it bothers us too much, we can find an organization for them later